pkg/cmd/tasks/gentypes/gentypes.go

```diff
@@ -0,0 +1,140 @@
+package gentypes
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/airplanedev/cli/cmd/airplane/auth/login"
+	"github.com/airplanedev/cli/pkg/api"
+	"github.com/airplanedev/cli/pkg/cli"
+	"github.com/airplanedev/cli/pkg/logger"
+	"github.com/airplanedev/cli/pkg/utils"
+	libapi "github.com/airplanedev/lib/pkg/api"
+	"github.com/airplanedev/lib/pkg/build"
+	"github.com/airplanedev/lib/pkg/runtime/typescript"
+	"github.com/airplanedev/lib/pkg/utils/fsx"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+type config struct {
+	root     *cli.Config
+	taskSlug string
+	envSlug  string
+	fileName string
+}
+
+func New(c *cli.Config) *cobra.Command {
+	var cfg = config{root: c}
+
+	cmd := &cobra.Command{
+		Use:   "gentypes slug",
+		Short: "Generates types for Node.js task parameters",
```

Comment:
Would be nice to support Python as it has types built-in nowadays (cc @ameyakhare). For now, let's remove the reference to Node.js here and assume we'll add a Python implementation at some point.

```suggestion
		Short: "Generates types for task parameters",
```